webserver/pkgpkr/webservice/templates/webservice/recommendations.html

```diff
@@ -1,19 +1,61 @@
 {% extends "webservice/base.html" %}
 
 {% block title %}
-    Package Picker
+    Recommendations
 {% endblock %}
 
 {% block body %}
-
-    <h3>Recommendations for `{{ repository_name }}`:</h3>
-
-    <ul>
-        {% for recommendation in recommendations %}
-
-            <li><a target="_blank" href="{{ recommendation.url }}">{{ recommendation.name }}
-                - {{ recommendation.average_downloads }} - {{ recommendation.keywords }} - {{ recommendation.rate  }}</a></li>
-        {% endfor %}
-    </ul>
-
+<section class="section">
+  <div class="container">
+    <div class="content is-medium">
+      <h2>RECOMMENDATIONS for `{{ repository_name }}`</h2>
+      <br>
+      <table>
+        <tr>
+          <td style="width:200px;">Selected Category: </td>
+          <td style="width:80px;"><input type="text" id="categoryName" name="categoryName" disabled style="border: none; background-color: white; font-size:20px"></td>
```

Comment:
Inline styles are generally not best practice. Let's stick these in `webserver/pkgpkr/webservice/templates/webservice/styles.css`.

Reply:
I see, I will try to provide id for elements instead and put its style into styles.css